combat-trainer.lic

```diff
@@ -3585,7 +3584,11 @@ class GameState
       attack_override
     elsif @use_weak_attacks
       brawling? ? 'gouge' : 'jab'
-    elsif @use_barb_combos
+    # @use_barb_combos needs to be removed given people time to switch
+    # !offhand is there because you can't combo while training offhand weapon without main hand
+    # !brawling is there because you can't use a weapon for brawling commands
+    # Both of these issues will cause problems with the combo array system
+    elsif @use_barb_combos || @use_analyze_combos && !offhand? && !brawling?
```

Comment:
:x: This needs parens: `(@use_barb_combos || @use_analyze_combos) && ... && ...`


```ruby
a = true
b = false
c = false
d = false
puts a || b && c && d
```

results in `true`

Reply:
Thought parens weren't a thing in ruby, lol, wasn't sure, thanks

Reply:
Ruby isn't magic? Got it.